node/hubController.js

```diff
@@ -289,7 +287,8 @@ class HubController {
     }
 
     _invokeMethod(translator, deviceInfo, methodName, params) {
-        console.log("----------------- _invokeMethod " + methodName + " with params "+ JSON.stringify(params));
+        this.ConsoleLogger.verbose(
+            `_invokeMethod ${methodName} with params ${JSON.stringify(params)}`);
```

Comment:
Ok so we arent consistent with this then? (To be clear this adds a space of 2 characters as whitespace? ) 

---
In reply to: [103828178](https://github.com/openT2T/cloud/pull/26#discussion_r103828178) [](ancestors = 103828178)